bridge/test/LibUtils_test.js

```diff
@@ -97,4 +97,24 @@ contract('LibUtils', async function (accounts) {
         });
     });
 
+    describe('toUint128', async function() {
+        it('Should convert bytes to uint128', async function() {
+            const number = 16;
+            const hex = number.toString(16);
+            const hex32BytesString = '0x' + hex.padStart(32, '0');
+
+            const returnValue = await this.utilsLib.toUint128(hex32BytesString, 0);
+
+            assert.equal(returnValue, number);
+
+            await truffleAssertions.passes(this.utilsLib.toUint128(hex32BytesString, 0));
```

Comment:
this line calls again the function while the line 106 does exactly the same. Shouldn't you just use the already retrieved converted value?